pandaserver/configurator/Configurator.py

```diff
@@ -842,7 +842,7 @@ def __init__(self, taskBuffer, log_stream=None):
         else:
             self.log_stream = _logger
 
-        if hasattr(panda_config, "CRIC_URL_SCHEDCONFIG"):
+        if hasattr(panda_config, "CRIC_URL_TAGS"):
```

Comment:
@fbarreir  Is this a desirable fix? Seems this line has been a typo since ever added.